java/lang/String/StringTest.cpp

```diff
@@ -481,4 +481,55 @@ TEST (JavaLang, StringSubString) {
 	result = subString.toString();
 	expect = (string) "ello ";
 	ASSERT_STR(expect, result);
+}
+
+TEST(JavaLang, StringFormat) {
+
+    int intValue = -123;
+    long longValue = 123;
+    float floatValue = 123.456;
+    double doubleValue = 123.456789;
+    string stringValue = "string";

```

Comment:
Sorry @sonda, I made a typing mistake. That must be cast, not catch.
@loint, confirmed.